content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -341,4 +341,4 @@ and post-upgrade manifest file for a certain component, a backup file for it wil
 `kubeadm upgrade node` does the following on worker nodes:
 
 - Fetches the kubeadm `ClusterConfiguration` from the cluster.
-- Upgrades the kubelet configuration for this node.
+- Upgrades the kubelet configuration for this node.
```

Comment:
please preserve the empty new line at the end of the document.

xref https://github.com/kubernetes/website/issues/25732

Reply:
Done, see [commit](https://github.com/kubernetes/website/pull/25794/commits/effb95d672f0af4af148722a105949bc7156b36b)